proto/regen/ecocredit/v1/tx.proto

```diff
@@ -321,6 +324,9 @@ message MsgCancel {
   // credits are the credits being cancelled.
   repeated CancelCredits credits = 2;
 
+  // reason is any arbitary encoded data, will be passed to an event.
+  string reason = 3;
```

Comment:
As mentioned above, this will be more generic given we decided to add a bridge specific event.

```suggestion
  // reason is any arbitrary string that specifies the reason for cancelling credits.
  string reason = 3;
```